zebra-chain/src/transparent/serialize.rs

```diff
@@ -224,9 +222,11 @@ impl ZcashSerialize for Output {
 
 impl ZcashDeserialize for Output {
     fn zcash_deserialize<R: io::Read>(mut reader: R) -> Result<Self, SerializationError> {
+        let reader = &mut reader;
+
         Ok(Output {
-            value: reader.read_u64::<LittleEndian>()?.try_into()?,
```

Comment:
same here

Reply:
I think this is a bitcoin encoding, so it's not part of the Zcash spec.

But I assume so. (Unless this is the Sapling value balance field, but that's unlikely, because this is a transparent transaction.)